ffi/record_meta.c

```diff
@@ -8,10 +8,12 @@
 #define EXPORT
 #endif
 
-typedef struct meta_t {
+/* TODO: tis is replicated in ffi_platypus.h, which is bad */
+typedef struct _ffi_pl_record_meta_t {
   ffi_type top;
```

Comment:
This was renamed to ffi_type in `ffi_platypus.h` and will need updating where used in this .c file too.